src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise.route.ts

```diff
@@ -12,10 +12,12 @@ import { Observable } from 'rxjs';
 import { HttpResponse } from '@angular/common/http';
 import { filter, map } from 'rxjs/operators';
 import { Course } from 'app/entities/course.model';
+import { ExerciseGroup } from 'app/entities/exercise-group.model';
+import { ExerciseGroupService } from 'app/exam/manage/exercise-groups/exercise-group.service';
 
 @Injectable({ providedIn: 'root' })
 export class TextExerciseResolver implements Resolve<TextExercise> {
-    constructor(private textExerciseService: TextExerciseService, private courseService: CourseManagementService) {}
+    constructor(private textExerciseService: TextExerciseService, private courseService: CourseManagementService, public exerciseGroupService: ExerciseGroupService) {}
```

Comment:
Why is `exerciseGroupService` public here?

Reply:
Fixed